### DISCUSSION TITLE
Error while Deploying

### DISCUSSION THREAD

**Discussion by SairajMN** [Cyfrin/foundry-full-course-cu#1941] [Q&A] [answered]
![Screenshot 2024-07-03 204859](https://github.com/Cyfrin/foundry-full-course-cu/assets/112056445/4e0ee370-49c0-4df8-9114-fcf1f1bb3141)
 Error in Deploying the SimpleStorage.sol with zkSync pulgin, the complilation of code went well but the Deploy button is not visible (I'm unable to deploy it).

**Comment by EngrPips** [comment, marked as answer]
Hello @SairajMN, Here [`https://github.com/Cyfrin/foundry-full-course-cu/discussions/1902`](https://github.com/Cyfrin/foundry-full-course-cu/discussions/1904#discussion-6875228) is a discussion for a similar issue. You should read through the discussion, and you should find a way out
